internal/pkg/cli/env_upgrade.go

```diff
@@ -4,9 +4,23 @@
 package cli
 
 import (
+	"errors"
+	"fmt"
+
+	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/term/prompt"
+	"github.com/aws/copilot-cli/internal/pkg/term/selector"
 	"github.com/spf13/cobra"
 )
 
+const (
+	envUpgradeAppPrompt = "In which application is your environment?"
+
+	envUpgradeEnvPrompt = "Which environment do you want to upgrade?"
+	envUpgradeEnvHelp   = `Upgrades the AWS CloudFormation template for your environment
+to support latest Copilot features.`
```

Comment:
The nitpickiest nit: "support **the** latest" because you have "the" in "Upgrades the AWS CloudFormation..."?

Reply:
Done!